src/content/en/updates/_shared/deprecations.html

```diff
@@ -0,0 +1 @@
+Chrome Platform Status provides a list of [current deprecations](https://www.chromestatus.com/features#browsers.chrome.status:%22Deprecated%22).
```

Comment:
You cannot use markdown in an `.html` file, you'll need to change this to true HTML